WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -40,6 +54,11 @@ class PageListViewModel @Inject constructor() : ViewModel() {
 
     private lateinit var pagesViewModel: PagesViewModel
 
+    private val featuredImageMap = HashMap<Long, String>()
+
+    val site: SiteModel
```

Comment:
Could we avoid exposing the `SiteModel` to the Fragment/Activity? I think it's only used to calculate `isPhotonCapable` which could be done in the `ViewModel` instead and exposed as a lazy(?) value.

We could also consider making `val pages: LiveData<List<PageItem>> = _pages` to be a `Pair` of `List<PageItem>` and the `isPhotonCapable`, so it's only exposed when it's needed and always calculated just before it's necessary. When we need more data in `PageListFragment.setPages`, we could make that a `Pair` of `List<PageItem>` and a `Config` object where we hold information such as `isPhotonCapable`.

Reply:
Interesting idea, I'll give it a try.